e2e/preview.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { test, expect } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(
+    '/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-1'
+  );
+  await page.keyboard.press('Escape');
```

Comment:
Why do we need this?

Reply:
When the page loads, a dialog appears which shows the final preview of the task. We need to close the dialog to click on Code Editor to change the code, so i thought i should close the dialog in the beginning as it would show the preview component clearly and won't interfere with other test cases (if we decide to add more test cases).

Reply:
We could avoid that by visiting step 2 instead of step 1. Either that, or we should add a comment for this line.

Reply:
Moved to step 2 to avoid pressing Escape key.